### PR TITLE
v1.9.1

### PR DESCRIPTION
**Pull request by smlindauer**
Various bug fixes and patches for 1.9.0

### CODE REVIEW

File: src/sasctl/pzmm/pickle_model.py [hunk 1]
```diff
@@ -20,8 +25,8 @@ class PickleModel:
     @classmethod
     def pickle_trained_model(
         cls,
-        trained_model,
-        model_prefix,
```
Comment (djm21) [inline src/sasctl/pzmm/pickle_model.py:24]: Any reason why the order of these two were switched?

File: src/sasctl/pzmm/pickle_model.py [hunk 2]
```diff
@@ -106,22 +110,29 @@ def pickle_trained_model(
...
+                        "The h2o package is required to save the model as a binary h2o"
+                        "model."
+                    )
+                h2o.save_model(
+                    model=trained_model,
+                    force=True,
+                    path=str(pickle_path),
+                    filename=f"{model_prefix}.pickle",
+                )
             # For MOJO H2O models, gzip the model file and adjust the file extension
```
Comment (djm21) [inline src/sasctl/pzmm/pickle_model.py:126]: This comment is no longer accurate

**Review by djm21 — COMMENTED** [review]
Nothing major to look at, just a few minor tweaks

File: src/sasctl/pzmm/pickle_model.py (hunk 1 above)
Comment (smlindauer) [reply src/sasctl/pzmm/pickle_model.py:24]: `trained_model` is optional in the mlflow case, so I shifted it behind the required `model_prefix` argument.

File: examples/pzmm_h2o_model_import.ipynb [hunk 3]
```diff
@@ -164,17 +164,17 @@
...
        "<tr><td>H2O_data_parsing_timezone:</td>\n",
        "<td>UTC</td></tr>\n",
        "<tr><td>H2O_cluster_version:</td>\n",
        "<td>3.38.0.4</td></tr>\n",
        "<tr><td>H2O_cluster_version_age:</td>\n",
-       "<td>2 months and 24 days </td></tr>\n",
+       "<td>3 months and 26 days !!!</td></tr>\n",
        "<tr><td>H2O_cluster_name:</td>\n",
-       "<td>H2O_from_python</td></tr>\n",
+       "<td>H2O_from_python_sclind_ogpklg</td></tr>\n",
```
[the remark this answers is not in the document]
Comment (smlindauer) [reply examples/pzmm_h2o_model_import.ipynb:177]: Good catch.

File: examples/pzmm_h2o_model_import.ipynb [hunk 4]
```diff
@@ -2266,28 +2266,25 @@
...
-     "data": {
-      "text/plain": [
-       "'~\\\\data\\\\hmeqModels\\\\H2OBinaryGLM\\\\glmfit'"
-      ]
-     },
-     "execution_count": 9,
-     "metadata": {},
-     "output_type": "execute_result"
+     "ename": "H2OResponseError",
+     "evalue": "Server error water.exceptions.H2OIllegalArgumentException:\n  Error: Illegal argument: dir of function: exportModel: water.api.FSIOException: FS IO Failure: \n accessed path : file:/C:/Users/sclind/Documents/Python%20Scripts/GitHub/sassoftware/python-sasctl/examples/data/hmeqModels/H2OBinaryGLM/glmfit msg: File already exists\n  Request: GET /99/Models.bin/glmfit\n    params: {'dir': 'data/hmeqModels/H2OBinaryGLM\\\\glmfit', 'force': 'False', 'export_cross_validation_predictions': 'False'}\n",
```
[the remark this answers is not in the document]
Comment (smlindauer) [reply examples/pzmm_h2o_model_import.ipynb:2276]: Yep, scrubbed the json representation of the notebook.

File: src/sasctl/pzmm/pickle_model.py (hunk 2 above)
Comment (smlindauer) [reply src/sasctl/pzmm/pickle_model.py:126]: Fixed.

**Pull request merged**